db/schema.rb

```diff
@@ -126,10 +126,14 @@
   create_table "user_settings", force: :cascade, options: "ENGINE=InnoDB DEFAULT CHARSET=utf8" do |t|
     t.boolean  "receive_notification",            default: true
     t.integer  "unread_notification"
-    t.float    "radius_display",       limit: 24, default: 5.0
+    t.boolean  "favorite_location",               default: false
+    t.string   "favorite_address"
+    t.float    "favorite_latitude",    limit: 24
+    t.float    "favorite_longitude",   limit: 24
+    t.integer  "radius_display",                  default: 5
     t.integer  "user_id"
-    t.datetime "created_at",                                     null: false
-    t.datetime "updated_at",                                     null: false
+    t.datetime "created_at",                                      null: false
+    t.datetime "updated_at",                                      null: false
```

Comment:
**Reporter**: CHECKSTYLE
**Rule**: com.puppycrawl.tools.checkstyle.Metrics/LineLength
**Severity**: INFO
**File**: db/schema.rb L136

Line is too long. [99/80]

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>*